### PR TITLE
Create operator resource file for WKO target

### PR DESCRIPTION
**Pull request by rakillen**
Expose template files in lib/targets directory.
Please suggest any relevant comments to add to binding.yaml, model.yaml templates.

Documentation will be in a separate PR.

Internal Jira WDT-461

### CODE REVIEW

File: core/src/main/python/wlsdeploy/tool/util/targets/additional_output_helper.py [hunk 1]
```diff
@@ -43,42 +42,46 @@
...
-    :param model_context: used to determine location and content for the configurations
+    Create and write additional output for the configured target type.
+    :param model: Model object, used to derive some values in the output
+    :param model_context: used to determine location and content for the output
     :param aliases: used to derive secret names
     :param exception_type: the type of exception to throw if needed
     """
 
     # -output_dir argument was previously verified
     output_dir = model_context.get_kubernetes_output_dir()
```
Comment (CarolynRountree) [inline core/src/main/python/wlsdeploy/tool/util/targets/additional_output_helper.py:55]: could model_context combine the two output_dir into one common routine?

File: core/src/main/targetconfigs/vz/model.yaml [hunk 2]
```diff
@@ -25,6 +36,7 @@ spec:
             replicas: {{{replicas}}}
```
Comment (CarolynRountree) [inline core/src/main/targetconfigs/vz/model.yaml:36]: what is replicas ?

**Review by CarolynRountree — APPROVED** [review]

File: core/src/main/python/wlsdeploy/tool/util/targets/additional_output_helper.py (hunk 1 above)
Comment (rakillen) [reply core/src/main/python/wlsdeploy/tool/util/targets/additional_output_helper.py]: Done, combined into get_output_dir()

File: core/src/main/targetconfigs/vz/model.yaml (hunk 2 above)
Comment (rakillen) [reply core/src/main/targetconfigs/vz/model.yaml:38]: Added this comment to both model.yaml files:
```
# The number of cluster member Managed Server instances to start for this WebLogic cluster
```

**Review by ddsharpe — APPROVED** [review]
LGTM

**Pull request merged**